bin/foreman-tasks

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env ruby
```

Comment:
The idea is to remove it from foreman-tasks and move it here - so no - in fact it should have the same name in my opinion, to make sure it plays well with existing tooling that expects it